galaxy.yml

```diff
@@ -11,8 +11,8 @@ authors:
     - Tom Page @Tompage1994
     - Sean Sullivan @sean-m-sullivan
     - David Danielsson @djdanielsson
-repository: https://github.com/redhat-cop/controller_configuration/
-issues: https://github.com/redhat-cop/controller_configuration//issues
+repository: https://github.com/redhat-cop/infra.controller_configuration/
+issues: https://github.com/redhat-cop/infra.controller_configuration//issues
```

Comment:
it looks like a double slash at some point got put in here, while you are at it could you fix that?

Reply:
@djdanielsson Thank you for reviewing. I fixed it.